### DISCUSSION TITLE
install script

### DISCUSSION THREAD

**Discussion by s03i0path** [end-4/dots-hyprland#119] [Ideas]
hey man, can you make a install script for this dotfiles?

**Comment by end-4** [comment]
thank you for reminding my lazy ass

**Reply by s03i0path** [reply]
come on my friend.

**Reply by end-4** [reply]
Done https://github.com/end-4/dots-hyprland/commit/ab30590bd409981c5b4e3abdcbab87c8d4c8ef5c
I'm going to appreciate it if you can test it

**Reply by end-4** [reply]
oh and that's only for illogical-impulse as it's the only functioning one that i care about

**Comment by MarcoBI24** [comment]
AMIGOS AYUDENME>
corri el archivo ./install.sh y malogro todo mi kali linux
tenia instalado el hyprland y queria que se viera como el hyprland de este repositorio, asi que 
descargue toda la rama Illillogical-impulse y corri el archivo.

**Reply by end-4** [reply]
Kali is Debian-based, so the script won't work properly.
Can you clarify what was ruined?

**Comment by clsty** [comment]
The install script is highly usable now. Read the [docs](https://end-4.github.io/dots-hyprland-wiki/en/i-i/01setup) for more.

Closing.
